tests/e2e/screenObjects/inventoryItem.js

```diff
@@ -47,25 +50,28 @@ class InventoryItemScreen extends Base{
    *
    * @return {string}
    */
-  getSwagPrice(){
+  getSwagPrice() {
+    Gestures.scrollDownToElement(this.price, 4);
+
     return getTextOfElement(this.price);
   }
 
   /**
-   * Get the text of the selected swag
+   * Get the label text of the selected swag
    *
    * @return {string}
    */
-  getSwagItemText(){
-    return `${this.getSwagDescription()} ${this.getSwagPrice()}`;
+  getSwagItemLabelText() {
+    return this.getSwagDescription();
```

Comment:
If this is just redirecting to getSwagDescription(), can we just remove this and reassign all call sites to call that function?

Reply:
Hmm, good call, will change that, tnx!